apps/sms/services/js/conversation/conversation_service.js

```diff
@@ -22,26 +38,74 @@ if (!('BridgeServiceMixin' in self)) {
   ]);
 
   var ConversationService = {
+    mobileMessageClient: null,
+
     /**
      * Initializes our service using the Service mixin's initService.
      */
     init() {
       this.initService();
+
+      this.mobileMessageClient = bridge.client(
+        'mozMobileMessageShim',
+        new BroadcastChannel('mobile-message-shim-channel')
```

Comment:
Yeah I think it's good for now, btw I'm going to change the message shim name(https://github.com/steveck-chung/gaia/commit/848b3f9bc14b07c861afa35bec041fb4df3b568a) to replace the camel case with dash to sync with our service naming.